_docs/kyc.md

```diff
@@ -0,0 +1,55 @@
+---
+title: KYC requirements
+permalink: /docs/kyc/
+---
+
+* Table of contents
+{:toc}
+
+Rules and regulations in the markets we operate require that all senders went through a KYC process before being allowed to transact funds. This is regardless whether our system is used for payout or for collection purposes. When you sign up with us our Compliance team will assess your KYC processess and based on that decide on how you'll be able to use our site. Based on their assessment there are three categories:
+
+* We don't need to check or approve your senders before they are able to transact
+* We need to check and approve your senders before they are able to transact
+* We need to ask for extra documents from your senders, and then check and approve them before they are able to transact
+
+In this document we'll go through these options and what they mean for using our site:
+
+# No checks required
+
+If you are already doing a full KYC on your customers then we won't need to re-do them. In this case any sender you create in our system will immediately become `approved` and can immediately be used for both payments and collections.
```

Comment:
As per the convo in IDT channel we may want to reword this so that it's clear the sender will stay in the initial state but will be considered as approved.

Reply:
Added a not under this section